### PR TITLE
A user can edit details of a bedspace

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

* A user can edit the details of a bedspace

This PR begins with some minor tidying to existing integration/E2E tests

## Screenshots of UI changes

localhost_3000_properties_f99b65ed-e2bf-46a6-81f8-a95a2c2a4d07_bedspaces_c8fe9f85-d577-4217-908f-b9cc99e4b33c

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[A user can edit details of a bedspace](https://trello.com/c/0SfeOMYF/502-a-user-can-edit-details-of-a-bedspace)

File: integration_tests/mockApis/room.ts [hunk 1]
```diff
@@ -54,4 +54,27 @@ export default {
...
   stubRoomCreateErrors: (args: { premisesId: string; params: Array<string> }): SuperAgentRequest =>
     stubFor(errorStub(args.params, paths.premises.rooms.create({ premisesId: args.premisesId }), 'POST')),
+  stubRoomUpdate: (args: { premisesId: string; room: Room }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'PUT',
+        url: paths.premises.rooms.update({ premisesId: args.premisesId, roomId: args.room.id }),
+      },
+      response: {
+        status: 201,
```
Comment (richpjames) [inline integration_tests/mockApis/room.ts]: Should this be a 200?

File: wiremock/roomStub.ts [hunk 2]
```diff
@@ -49,10 +55,25 @@ rooms.push({
...
 
-const requiredFields = getCombinations(['name'])
+rooms.push({
+  priority: 99,
+  request: {
+    method: 'PUT',
+    urlPathPattern: `/premises/${guidRegex}/rooms/${guidRegex}`,
+  },
+  response: {
+    status: 201,
```
Comment (richpjames) [inline wiremock/roomStub.ts]: flagging this as well

**Review by richpjames — APPROVED** [review]
LGTM 
One minor thing: I think the commit message for a commit you squashed into e8cc6e945a4131da9f980eeddf44da4bbd6775a4 is still included

**Pull request merged**